src/ZipStream.php

```diff
@@ -57,7 +57,7 @@
  */
 class ZipStream
 {
-    const ZIP_VERSION_MADE_BY = 0x031E; // 3.00 on Unix
+    const ZIP_VERSION_MADE_BY = (6 << 8) + 3; // 3.00 on Unix
```

Comment:
Which is `0x603` in hex.

From https://www.iana.org/assignments/media-types/application/zip
```
      version made by (2 bytes)

	  The upper byte indicates the host system (OS) for the
	  file.  Software can use this information to determine
	  the line record format for text files etc.  The current
	  mappings are:

	  0 - MS-DOS and OS/2 (F.A.T. file systems)
	  1 - Amiga                     2 - VAX/VMS
	  3 - *nix                      4 - VM/CMS
	  5 - Atari ST                  6 - OS/2 H.P.F.S.
	  7 - Macintosh                 8 - Z-System
	  9 - CP/M                      10 thru 255 - unused

	  The lower byte indicates the version number of the
	  software used to encode the file.  The value/10
	  indicates the major version number, and the value
	  mod 10 is the minor version number.
```

This is a bit more complicated that it seems for example in [libzip source code](https://github.com/nih-at/libzip/blob/4c612cb79672016c0f077b9c511e77df82b35023/lib/zip_file_set_external_attributes.c#L37-L82
):
```c
zip_file_set_external_attributes(zip_t *za, zip_uint64_t idx, zip_flags_t flags, zip_uint8_t opsys, zip_uint32_t attributes) {
    zip_entry_t *e;
    int changed;
    zip_uint8_t unchanged_opsys;
    zip_uint32_t unchanged_attributes;


    if (_zip_get_dirent(za, idx, 0, NULL) == NULL)
	return -1;


    if (ZIP_IS_RDONLY(za)) {
	zip_error_set(&za->error, ZIP_ER_RDONLY, 0);
	return -1;
    }


    e = za->entry + idx;


    unchanged_opsys = (e->orig ? (zip_uint8_t)(e->orig->version_madeby >> 8) : (zip_uint8_t)ZIP_OPSYS_DEFAULT);
    unchanged_attributes = e->orig ? e->orig->ext_attrib : ZIP_EXT_ATTRIB_DEFAULT;


    changed = (opsys != unchanged_opsys || attributes != unchanged_attributes);


    if (changed) {
	if (e->changes == NULL) {
	    if ((e->changes = _zip_dirent_clone(e->orig)) == NULL) {
		zip_error_set(&za->error, ZIP_ER_MEMORY, 0);
		return -1;
	    }
	}
	e->changes->version_madeby = (zip_uint16_t)((opsys << 8) | (e->changes->version_madeby & 0xff));
	e->changes->ext_attrib = attributes;
	e->changes->changed |= ZIP_DIRENT_ATTRIBUTES;
    }
    else if (e->changes) {
	e->changes->changed &= ~ZIP_DIRENT_ATTRIBUTES;
	if (e->changes->changed == 0) {
	    _zip_dirent_free(e->changes);
	    e->changes = NULL;
	}
	else {
	    e->changes->version_madeby = (zip_uint16_t)((unchanged_opsys << 8) | (e->changes->version_madeby & 0xff));
	    e->changes->ext_attrib = unchanged_attributes;
	}
    }


    return 0;
}
```